### PR TITLE
Small bug fix on potential underflow error in constructBridgeOperations

### PR DESCRIPTION
**Pull request by cwang25**
- Noticed in one clause it does direct subtraction, which will cause underflow error if the chain has less than intended amount to bridge.

### CODE REVIEW

File: .gas-snapshot
Comment (kevincheng96) [inline .gas-snapshot:1]: We should probably just turn off gas snapshots for files in `src/builder`.

File: test/builder/QuarkBuilderCometWithdraw.t.sol [hunk 1]
```diff
@@ -496,4 +496,19 @@ contract QuarkBuilderCometWithdrawTest is Test, QuarkBuilderTest {
...
         );
     }
+
+    function testCometWithdrawCostTooHigh() public {
+        PaymentInfo.PaymentMaxCost[] memory maxCosts = new PaymentInfo.PaymentMaxCost[](2);
+        maxCosts[0] = PaymentInfo.PaymentMaxCost({chainId: 1, amount: 5e6});
+        maxCosts[1] = PaymentInfo.PaymentMaxCost({chainId: 8453, amount: 5e6});
+        QuarkBuilder builder = new QuarkBuilder();
+
+        vm.expectRevert(abi.encodeWithSelector(Actions.NotEnoughFundsToBridge.selector, "usdc", 4e6, 4e6));
```
Comment (kevincheng96) [inline test/builder/QuarkBuilderCometWithdraw.t.sol:506]: Shouldn't this be "USDC"? (capitalized)

**Review by kevincheng96 — APPROVED** [review]

**Comment by kevincheng96** [comment]
Good catch

File: test/builder/QuarkBuilderCometWithdraw.t.sol (hunk 1 above)
Comment (cwang25) [reply test/builder/QuarkBuilderCometWithdraw.t.sol:506]: That's good question... haha. 
I agree.. it should be capitalized,  let me double check... looks like all existing tests is verifying it with lower cased 😮 ...

Comment (kevincheng96) [reply test/builder/QuarkBuilderCometWithdraw.t.sol]: Weird that the check is case insensitive 🤔

Comment (cwang25) [reply test/builder/QuarkBuilderCometWithdraw.t.sol:506]: oh wow... it's case sensitive...  hmm I think I need to dig a bit deeper to see why some tests seems still carrying side effect of `toLowerCase()` and some don't.

**Pull request merged**